CONTRIBUTING.md

```diff
@@ -36,6 +43,21 @@ appropriate permissions.
 
 We use Sphinx for manual and automatic API [documentation](docs).
 
+### Contribution Release Note Exceptions
+When a new contribution is fixing a bug or minor issue with something has been
+recently contributed, it may be the case that no additional release notes are
+needed that wouldn't add redundancy to the document.
```

Comment:
Minor edit:

```suggestion
When a new contribution is fixing a bug or minor issue with something that has
been recently contributed, it may be the case that no additional release notes
are needed since they would add redundancy to the document.
```